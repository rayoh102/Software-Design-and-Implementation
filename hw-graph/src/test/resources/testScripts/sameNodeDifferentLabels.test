# Tests if we can connect between the same node with different labels

# Create a graph
CreateGraph graph1

# Add two nodes
AddNode graph1 n1
AddNode graph1 n2

# Add an edge
AddEdge graph1 n1 n2 e1

# Add same edge, different label
AddEdge graph1 n1 n2 e2

# Check edges
ListChildren graph1 n1