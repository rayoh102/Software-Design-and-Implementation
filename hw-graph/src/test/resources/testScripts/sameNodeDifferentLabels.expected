# Tests if we can connect between the same node with different labels

# Create a graph
created graph graph1

# Add two nodes
added node n1 to graph1
added node n2 to graph1

# Add an edge
added edge e1 from n1 to n2 in graph1

# Add same edge, different label
added edge e2 from n1 to n2 in graph1

# Check edges
the children of n1 in graph1 are: n2(e1) n2(e2)